tests/app/celery/test_letters_pdf_tasks.py

```diff
@@ -327,24 +327,25 @@ def test_get_key_and_size_of_letters_to_be_sent_to_print_handles_file_not_found(
 
 @mock_s3
 @pytest.mark.parametrize(
-    "print_run_deadline_utc, expected_time_called_with",
+    "frozen_time, expected_time_called_with",
     [
-        ("2021-06-01 12:00:00", datetime(2021, 6, 1, 13, 0, 0)),
-        ("2021-01-01 12:00:00", datetime(2021, 1, 1, 12, 0, 0)),
-        (None, datetime(2021, 6, 1, 17, 30)),
+        # 5:50pm and 5:30pm in GMT, represented in UTC
+        ("2021-01-01 17:50:00+00:00", "2021-01-01T17:30:00"),
+        # 5:50pm and 5:30pm in BST, represented in UTC
+        ("2021-06-01 16:50:00+00:00", "2021-06-01T16:30:00"),
     ],
 )
 def test_check_time_to_collate_letters_respects_print_run_deadline(
```

Comment:
minor nitpick - the other test of this function - `test_check_time_to_collate_letters_exits_early_outside_of_expected_window` - does a lot of unnecessary setup that looks like a relic from when it was testing a function with wider scope - can we get rid of some of that? (and also ideally colocate the three tests within the file, whether in a class or not i don't mind)